Tests/Anonymizer/Type/FixedValueAnonymizerTest.php

```diff
@@ -0,0 +1,160 @@
+<?php
+/**
+ * This file is part of the GDPR bundle.
+ *
+ * @category  Bundle
+ * @package   Gdpr
+ * @author    SuperBrave <info@superbrave.nl>
+ * @copyright 2018 SuperBrave <info@superbrave.nl>
+ * @license   https://github.com/superbrave/gdpr-bundle/blob/master/LICENSE MIT
+ * @link      https://www.superbrave.nl/
+ */
+
+namespace SuperBrave\GdprBundle\Tests\Anonymizer;
+
+use PHPUnit_Framework_MockObject_MockObject;
+use SuperBrave\GdprBundle\Anonymize\Type\FixedValueAnonymizer;
+use SuperBrave\GdprBundle\Manipulator\PropertyManipulator;
+use SuperBrave\GdprBundle\Tests\AnnotatedMock;
+use Symfony\Component\OptionsResolver\Exception\MissingOptionsException;
+use Symfony\Component\PropertyAccess\PropertyAccess;
+
+/**
+ * Tests the behaviour of the EmailAnonymizer
+ */
+class FixedValueAnonymizerTest extends \PHPUnit_Framework_TestCase
```

Comment:
If the service is going to be renamed to `FixedAnonymizer` then please rename this class too.

Reply:
@TomSchillemans